tests/cases/fourslash/refactorIntroduceDestruction34.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference path='fourslash.ts' />
+
+//// const item = {
+////     a: 1, b: 2
+//// }
+//// item./*a*/a/*b*/

```

Comment:
It seems like offering this refactor when it would destructure to one property that is only accessed once is overkill. I would think the refactor should only start being offered when it would destructure to more than one property
```ts
console.log(item.a);
console.log(item.b);
// =>
const { a, b } = item;
console.log(a):
console.log(b):
```
Or when the property is accessed more than once
```ts
console.log(item.a);
console.log(item.a);
// =>
const { a } = item;
console.log(a);
console.log(a);
```

If `triggerReason` is `"invoked"` this condition could be ignored as we would be asking for refactors "aggressively".